common/cache/src/main/java/org/thingsboard/server/cache/RedisTbTransactionalCache.java

```diff
@@ -57,6 +56,7 @@ public abstract class RedisTbTransactionalCache<K extends Serializable, V extend
     private final TbRedisSerializer<K, V> valueSerializer;
     private final Expiration evictExpiration;
     private final Expiration cacheTtl;
+    private final boolean cacheEnabled;
```

Comment:
If possible, let's add `RedisTbTransactionalCacheTest` where we will verify that the mock RedisConnection is not used when the cache is disabled.